protocol/contracts/libraries/Silo/LibWhitelist.sol

```diff
@@ -204,7 +204,7 @@ library LibWhitelist {
 
         // stalkEarnedPerSeason is set to int32 before casting down.
         s.sys.silo.assetSettings[token].deltaStalkEarnedPerSeason = (int32(stalkEarnedPerSeason) -
-            int32(s.sys.silo.assetSettings[token].stalkEarnedPerSeason)).toInt24();
+            int32(s.sys.silo.assetSettings[token].stalkEarnedPerSeason)).toInt32();
```

Comment:
surrounding the `s.sys.silo.assetSettings[token].stalkEarnedPerSeason` with a int32 and casting down to a toInt32 is unnecessary.

Reply:
Fixed: https://github.com/BeanstalkFarms/Beanstalk/pull/1092/commits/9249efddefd2f3013c10e468ab03c4b4b1e91ad9#diff-dc7f60cc6e63597b30e6d00ab9e69f139cdc59411ff3113e7a54d7b331b3d1ebR207